examples/python/beeai-chat/agent.py

```diff
@@ -26,28 +28,34 @@ def to_framework_message(role: str, content: str) -> beeai_framework.backend.Mes
             raise ValueError(f"Unsupported role {role}")
 
 
-@server.agent()
+@server.agent(
+    metadata=Metadata(
+        annotations=Annotations(
+            beeai_ui=PlatformUIAnnotation(
+                ui_type=PlatformUIType.CHAT,
+                user_greeting="Let's chat!",
+                display_name="Chat Agent",
+            )
+        )
```

Comment:
This has very little benefit over a simple dictionary (basically just IntelliSense) at the cost of introducing more of the platform to the ACP.